cats/static/cats/css/style.css

```diff
@@ -0,0 +1,3 @@
+ul li a:hover{
```

Comment:
Please avoid styling by html tags, it is only acceptable when you have to style custom element without possibility to add class to it, you don't have full control while styling like that
https://javascript.info/styles-and-classes